src/examples/Svg3.cpp

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
I think you can add svg resource to images and test with Svg example.
This example code itself doesn't have any special showcase code.

Reply:
Ok